records.go

```diff
@@ -165,19 +165,30 @@ const (
 	RRTypeWKS RRType = "WKS"
 )
 
+// Define a function to create a comment option
```

Comment:
It's good practice to start the comment with the function name (`WithComments defines a function to ...`).

Reply:
The  comment has been corrected